scissors.js

```diff
@@ -562,6 +562,11 @@ Command.prototype.propertyStream = function () {
  */
 Command.prototype._exec = function () {
   var stream = new Stream(), commands = this.commands.slice();
+    
+  stream.on('error', function (err) {
+    console.log(err);
```

Comment:
Can you change it to `console.error(err.message)` to make it consistent with the rest of the code? Thanks!

Reply:
Wait, I can do it here myself...